src/lib/utils/index.ts

```diff
@@ -73,3 +73,19 @@ export const RUST_ISO_UTC_DATE_FORMAT = 'yyyy-MM-dd HH:mm:ss';
 export function isTouchScreenOnly(): boolean {
   return window.matchMedia('(hover: none) and (pointer: coarse)').matches;
 }
+
+export function getCookie(cname: string) {
```

Comment:
nit: `cname` is a bit confusing (compare with e.g. cname lookup), and not really more helpful than just calling this `name`

Reply:
Agreed